tests/data_sources/optical_flow/test_optical_flow_data_source.py

```diff
@@ -0,0 +1,39 @@
+"""Test Optical Flow Data Source"""
+import tempfile
+
+import pytest
+
+from nowcasting_dataset.config.model import Configuration, InputData
+from nowcasting_dataset.data_sources.optical_flow.optical_flow_data_source import (
+    OpticalFlowDataSource,
+)
+from nowcasting_dataset.dataset.batch import Batch
+
```

Comment:
Nice tests!

It might be good to include a test that `OpticalFlowDataSource` moves pixels in the correct direction :slightly_smiling_face: (e.g. give it a pair of images where a small dot moves to the left, and check that the predicted images continue to move the dot to the left!) But that's probably slightly fiddly, so might be best left for a future PR :slightly_smiling_face:

Reply:
add issue for this?